setup.py

```diff
@@ -36,7 +36,7 @@ def recursive_include(relative_dir):
 
 setup(
     name='chalice',
-    version='1.24.1',
+    version='1.24.1+7',
```

Comment:
@hannes-ucsc What sets the build version number here? I see that `7` is the next number used based on the previous commits, but shouldn't the build number be reset every time the version is incremented?

Reply:
We set it. Both restarting and continuing the local version number are possible. The only requirement is that our version is considered newer than the upstream version and that we stick with our own precedent. We didn't reset the local version in the past so we shouldn't change that convention now.